modules/ROOT/pages/environment-limitations.adoc

```diff
@@ -0,0 +1,41 @@
+= Limitations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The following limitations exist for Runtime Fabric. These are important to understand when installing, configuring, 
+and managing Runtime Fabric.
+
+== Controller Nodes
+The maximum number of controller nodes supported is ???.
+
+== Worker Nodes
+The maximum number of worker nodes supported is 16.
+
+== Apps Per Worker Node
+The maximum number of apps that can be deployed per worker node is ???
+
+== Environments Per Runtime Fabric Instance???
+The maximum number of environments per Runtime Fabric instance is ??? 
```

Comment:
remove "instance"
@jeff-1amstudios what limit should we document here? Quickly testing, I was able to allocate 12 environments across 3 business groups to my prod RTF.